hardhat.config.ts

```diff
@@ -632,17 +635,17 @@ export default {
     },
     tenderly: {
       chainId: 3030, // using same avee deploy chainId on tenderly
-      url: `https://rpc.tenderly.co/fork/${process.env.TENDERLY_FORK_ID}`,
-      accounts: [process.env.BASE_TESTNET_PRIVATE_KEY as string],
+      url: '`https://rpc.tenderly.co/fork/${process.env.TENDERLY_FORK_ID}`',
+      accounts: [(process.env.BASE_TESTNET_PRIVATE_KEY as string) ?? 'BASE_TESTNET_PRIVATE_KEY'],
     },
-    // "base-mainnet": {
-    //   url: `https://base.gateway.tenderly.co/${process.env.TENDERLY_PROJECT_ID}`,
-    //   accounts: [process.env.BASE_PROD_PRIVATE_KEY as string],
-    //   gasPrice: 1000000000,
-    // },
-    'base-goerli': {
-      url: `https://base-goerli.gateway.tenderly.co/${process.env.TENDERLY_PROJECT_ID}`,
-      accounts: [process.env.BASE_TESTNET_PRIVATE_KEY as string],
+    base: {
+      url: 'https://rpc.ankr.com/base',
+      accounts: [(process.env.BASE_PROD_PRIVATE_KEY as string) ?? 'BASE_PROD_PRIVATE_KEY'],
+      gasPrice: 1000000000,
+    },
```

Comment:
Lets keep the old naming conventions as we have prepared deploy commands using those